pkg/features/versioned_kube_features.go

```diff
@@ -37,18 +37,22 @@ import (
 // Entries are alphabetized.
 var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate.VersionedSpecs{
 	AllowDNSOnlyNodeCSR: {
+		{Version: version.MustParse("1.0"), Default: true, PreRelease: featuregate.GA},
 		{Version: version.MustParse("1.31"), Default: false, PreRelease: featuregate.Deprecated},
 	},
 
 	AllowInsecureKubeletCertificateSigningRequests: {
+		{Version: version.MustParse("1.0"), Default: true, PreRelease: featuregate.GA},
```

Comment:
Should Deprecated feature explicitly indicate `LockToDefault=true` for 1.0 entries? Otherwise if we emulate something like 1.33 binVer on 1.31 emuVer, we could explicitly disable this feature by passing in the feature flag which was not an option available before.

Reply:
It would lock out use of the feature gate in emulated versions before the deprecation was introduced. Not locking is equivalent to backporting the deprecation gate toggle to all previous releases though, since emulation version should only give access to feature toggles available at a particular historical release.

Reply:
Locking to the default in earlier versions and then unlocking in current versions is weird. I don't think we should do that.